index.html

```diff
@@ -0,0 +1,125 @@
+              <li class="menulink"><a href="#join">Join</a></li>
+              <li class="menulink"><a href="#sponsor">Sponsor</a></li>
+              <li class="menulink"><a href="#news">News</a></li>
+              <button class="campaignBtn">DLGC Campaign</button>
+            </ul>
+          </nav>
+        </div>        
+      
+      <div class="bg-image">
+        <nav class="menuBars">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </nav>
+      </div>
+    </header>
+  
+    <section class="intro">
+      <h1>"Hello! Dog Lovers"</h1>
+      <div class="bold">
+        <h2>DOG LOVERS GLOBAL CONFERENCE <br> 2022</h2>
+      </div>
+      <div class="info">
+        <p>An interesting and delightful experience where dog experts from all over the world come around have a talk about our dogs feelings, their way of life and how you can become the most knowledgeable dog expert on the block.</p>
+      </div>
+      <div class="date">
+        <h3>2022.12.09(FRI) ~ 10(SAT)</h3>
+        <h4>@ Paw Royale Gallery, Banana Island, Nigeria</h4>
+      </div>
+    </section>
+  
+    <section class="program" id="program">
+      <h5>Main Program</h5>
+      <div class="bar-2"></div>
+      <div class="allprograms">
+          <article class="activities"><img class="logo" src="Images/present.svg" alt="lecture icon">
+            <h6>Lecture</h6><p>Listen to the speakers from various counrties about dog observation and lifestyle monitoring</p>
+          </article>
+          <article class="activities"><img class="logo" src="Images/dog.svg" alt="paw icon">
+            <h6>Exhibition</h6><p>Appreciate various kinds and quality of dog breeds of top dog breeders </p>
+          </article>
+          <article class="activities"><img class="logo" src="Images/forum.svg" alt="chat icon">
+            <h6>Forum</h6><p>Have the time to share your thoughts and opinions with experts for each topic</p>
+          </article>
+          <article class="activities"><img class="logo" src="Images/appraisal.svg" alt="scroll">
+            <h6>Workshop</h6><p>Try appraising your own dog going by the provided breed standard</p>
+          </article>
+          <article class="activities"><img class="logo" src="Images/meeting.svg" alt="meeting icon">
+            <h6>Connect</h6><p>Get oppurtunities to network with DL affiliates anf top dog breeders around the world</p>
+          </article>
+          <button class="joinBtn">Join DL Global Conference 2022</button> 
+      </div>  
+      <div class="lastbtn">
+        <a href="#SEE THE WHOLE PROGRAM">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h5>Featured Speakers</h5>
+      <div class="bar-2"></div>
+      <div class="featuredSpeakers" id="featuredSpeakers">
+      </div>
+      <button class="more">MORE <i class="fa-solid fa-chevron-down"></i></button>
```

Comment:
- [ ] (Optional) It would be nice if when clicked the `more` button showed all the speakers.
![image](https://user-images.githubusercontent.com/81932478/205330913-33e5d8a7-fc82-4f2b-a180-7b3f1053b4d9.png)